cmake/FindLpeg.cmake

```diff
@@ -10,5 +10,5 @@ ubuntu()
 find_package_handle_standard_args(Lpeg DEFAULT_MSG LPEG_LIBRARY)
 mark_as_advanced(LPEG_LIBRARY)
 
-add_library(lpeg INTERFACE)
-target_link_libraries(lpeg INTERFACE ${LPEG_LIBRARY})
+add_library(lpeg UNKNOWN IMPORTED)
```

Comment:
should we add a one line summary or something in a comment here? (`git blame` is a thing, but a comment that something is different from other library targets to work around a cmake heisenissue would be good)